build_tools/python/benchmark_suites/iree/module_execution_configs.py

```diff
@@ -14,29 +14,29 @@
     loader=iree_definitions.RuntimeLoader.EMBEDDED_ELF,
     driver=iree_definitions.RuntimeDriver.LOCAL_SYNC)
 
-ELF_CUDA_CONFIG = iree_definitions.ModuleExecutionConfig(
+CUDA_CONFIG = iree_definitions.ModuleExecutionConfig(
     id=unique_ids.IREE_MODULE_EXECUTION_CONFIG_CUDA,
     tags=["full-inference", "default-flags"],
-    loader=iree_definitions.RuntimeLoader.EMBEDDED_ELF,
+    loader=iree_definitions.RuntimeLoader.CUDA,
```

Comment:
There may be some terminology confusion here. Only the 'local' (CPU) HAL currently has a concept of "loader": https://github.com/iree-org/iree/tree/main/runtime/src/iree/hal/local/loaders. There isn't an equivalent concept for the other HAL drivers. Maybe these should be `RuntimeLoader.NONE` or `NOT_APPLICABLE`?

https://github.com/iree-org/iree/blob/a09e5644504542578fb79851b243a943af748687/CMakeLists.txt#L126-L159

Reply:
Oh, makes sense. Renamed to None.